MRN-Code/coinstac
# Redesign/consortia delete

* **Problem:**
   1. Consortium items were able to be created but not deleted
* **Solution:**
   1. Created a graphql mutation to accommodate deleting a consortium item 
   1. Added code in resolvers.js to delete consortium item from rethinkdb
   1. Deleted cached consortium item so that the front end 
   1. User permissions determine front-end access and back-end GraphQL access.
* **Tests:**
   1. Click on consortia tab on the left
   1. Create new consortium if one with the delete button does not exist
   1. Click on delete button to confirm that the consortium was deleted
